File: sellergoods-interfaceings/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService {

    List<TbBrand> findAll();

    PageResult  findPage(int num,int pageSize);

    void add(TbBrand tbBrand);

    void update(TbBrand tbBrand);

    TbBrand findOne(long id);

    void delete(long[] ids);

    PageResult findPage(TbBrand tbBrand,int pageNum,int pageSize);

    /**
     * 品牌下拉框数据
     */
    List<Map> selectOptionList();
}

File: user-webings/src/main/webapp/js/service/brandService.js
//品牌服务层
app.service('brandService',function ($http) {
    //读取列表数据绑定到表单中
    this.findAll = function () {
        return $http.get('../brand/findAll.do');
    }

    this.findPage = function (page,rows) {
        return $http.get('../brand/findPage.do?page='+page+'&rows='+rows);
    }

    this.add = function (entity) {
        return $http.put('../brand/add.do',entity)
    }

    this.update = function (entity) {
        return $http.put('../brand/update.do',entity)
    }

    this.findOne = function (id) {
        return $http.get('../brand/findOne.do?id='+id)
    }

    this.dele = function (selectIds) {
        return $http.get('../brand/delete.do?ids='+selectIds)
    }
    this.search = function(page,rows,searchEntity){
        return  $http.post('../brand/search.do?page='+page+'&rows='+rows,searchEntity)
    }

    this.selectOptionList = function () {
        return $http.get('../brand/selectOptionList.do');
    }
});
